### DISCUSSION TITLE
Commodity vs Industry Factors in v1.2

### DISCUSSION THREAD

**Discussion by jaygravityclimate** [USEPA/supply-chain-factors#42] [Q&A] [answered]
Hello all, I understand the latest release only has one data set, whereas prior releases had both industry and commodity factors. The guidance says the factors are primarily to be used for spend-based scope 3 categories 1 and 2 calculations. In the past, my understanding was that industry (not commodity) factors could be used for estimating scope 3 category 15 – i.e., estimating emissions based only on a company's revenue and industry. Is the single set of new factors suitable for measuring both spend-based purchases and revenue-based calculations?

- As an example for former: emissions per dollar spent in X industry.
- As an example for letter: emissions for a company in X industry based on its annual revenue.

**Comment by bl-young** [comment, marked as answer]
Correct - the latest release only provides results in commodity form, not industry. In general these should be sufficient for the calculations you describe. However you can also generate factors for an industry model by adjusting the [CommodityorIndustryType](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml#L10) parameter and re-running the factors.

**Reply by TheoC-ERM** [reply]
Dear Ben, thank you for the instructions provided. However, when I adjust this parameter and rerun the code to generate the Supply Chain Emission Factors tables, I encounter the following error message: 'Error in margins_by_sector[gsub("/.", "", colnames(A_margin)), gsub("/.", : subscript out of bounds.'
Do you have any ideas on how to solve this issue? Are there any additional lines or parameters that need adjustment?
Thanks for your assistance,

**Reply by bl-young** [reply]
I will take a look and track in #47

**Reply by WesIngwersen** [reply]
This error was resolved
